content/20.treatments.md

```diff
@@ -517,6 +516,43 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
 This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
 This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
 
+##### Probiotics
+
+Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” [@doi:10.1038/nrgastro.2014.66]. 
+As a result of the supposed modest benefits of probiotics against common viral infections and evidence that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411], it has been hypothesized that probiotics may have therapeutic benefits against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
+
+**Potential Mechanisms.**
+Probiotics and next-generation probiotics, which are more akin to pharmacological grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063]. 
+However, the mechanisms by which probiotics affect these various conditions are diverse, heterogeneous, and strain specific [@doi:10.1093/advances/nmy063]. 
+Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079]. 
+However, there is also a bi-directional relationship between the lungs and gut microbiota known as the gut-lung axis [@doi:10.1038/mi.2011.55], whereby gut microbial metabolites and endotoxins may affect the lungs via the circulatory system and the lung microbiota in return may affect the gut [@doi:10.1111/cmi.12966]. 
+Therefore, the gut-lung axis may play role in our future understanding of COVID-19 pathogenesis and become a target for probiotic treatments [@doi:10.1016/j.virusres.2020.198018].
+
+**Current Evidence.**
+Probiotics have been associated with the reduction of risk and duration of viral upper respiratory tract infections [@doi:10.1007/s10096-014-2086-y; @doi:10.1002/14651858.CD006895.pub3; @doi:10.1016/j.ijantimicag.2008.11.005]. 
+Meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection [@doi:10.1002/14651858.CD006895.pub3; @doi:10.1017/S0007114514000075]. 
```

Comment:
```suggestion
Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection [@doi:10.1002/14651858.CD006895.pub3; @doi:10.1017/S0007114514000075]. 
```
Suggested because it looks like [@doi:10.1016/j.ijantimicag.2008.11.005] evaluated but didn't find a change in incidence